packages/cashier/src/Containers/deposit.jsx

```diff
@@ -82,6 +83,9 @@ const Deposit = ({
     if (is_virtual) {
         return <Virtual />;
     }
+    if (isCryptocurrency(currency)) {
+        return <CryptoDeposit />;
+    }
```

Comment:
please check if there is system maintenance and if cashier/deposit is locked before showing this